src/tests/test_array.py

```diff
@@ -1125,7 +1125,7 @@ def test_md_array_conversion():
 
     for i in range(5):
         for n in range(5):
-            items.SetValue(Spam(str((i, n))), (i, n))
+            items.SetValue(Spam(str((i, n))), i, n)
```

Comment:
This looks like a breaking change to me. Why was this affected?

Reply:
I made a support for calling `foo(int x = 0, params object[] bar)` by `foo(10, 1, 2, 3)` just as the C# code like, this feature is just conflict with master (it needs call `foo(10, [1, 2, 3])`) which I thought it's not pretty enough.